apprentice-react-o-shop/src/components/NavBar/NavBar.js

```diff
@@ -1,5 +1,8 @@
 import React, { useState } from 'react';
 
+import LinkListItem from './ListLinkItem'
+import DropDownItem from './DropDownItem'
+
 function NavBar(){
 
     const [open, setOpen] = useState(false)
```

Comment:
Lets rename this state and hook to be more descriptive to what it is and what it's doing